frame/contracts/src/storage/meter.rs

```diff
@@ -447,21 +463,30 @@ where
 		// We need to make sure that the contract's account exists.
 		T::Currency::transfer(origin, contract, ed, Preservation::Preserve)?;
 
+		System::<T>::inc_consumers(&contract)?;
```

Comment:
Need a comment here why we increase consumers vs providers vs sufficients.

Reply:
This line of code already existed on the original code, it's just a rollback to what it already is on master. Anyway, I have added a comment in line with what you mentioned [here](https://github.com/paritytech/substrate/pull/14589#discussion_r1293523496)

Reply:
Thanks :)